molecule/model/schema_v2.py

```diff
@@ -710,6 +710,85 @@ def pre_validate_base_schema(env, keep_string):
     },
 }
 
+platforms_lxd_schema = {
+    'platforms': {
+        'type': 'list',
+        'schema': {
+            'type': 'dict',
+            'schema': {
+                'name': {
+                    'type': 'string',
+                },
+                'url': {
+                    'type': 'string',
+                },
+                'cert_file': {
+                    'type': 'string',
+                },
+                'key_file': {
+                    'type': 'string',
+                },
+                'trust_password': {
+                    'type': 'string',
+                },
+                'source': {
+                    'type': 'dict',
+                    'schema': {
+                        'type': {
+                            'type': 'string',
+                        },
+                        'mode': {
+                            'type': 'string',
+                            'allowed': [
+                                'pull',
+                                'local',
+                            ],
+                        },
+                        'server': {
+                            'type': 'string',
+                        },
+                        'protocol': {
+                            'type': 'string',
+                            'allowed': [
```

Comment:
I made the schema as specific as possible, these are the only values valid for the LXD api. There are now allowed values ([source](https://github.com/lxc/lxd/blob/master/doc/rest-api.md#post-1)).